src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs

```diff
@@ -610,14 +610,19 @@ public override PSHostUserInterface UI
         /// <summary>
         /// See base class.
         /// </summary>
-        public void PushRunspace(Runspace newRunspace)
+        public void PushRunspace(Runspace runspace)
         {
             if (_runspaceRef == null)
             {
                 return;
             }
 
-            RemoteRunspace remoteRunspace = newRunspace as RemoteRunspace;
+            RemoteRunspace remoteRunspace = runspace as RemoteRunspace;
+            if (remoteRunspace is null)
+            {
+                throw new ArgumentException(ConsoleHostStrings.PushRunspaceNotRemote, nameof(runspace));
+            }
+
             Dbg.Assert(remoteRunspace != null, "Expected remoteRunspace != null");
```

Comment:
The check is not need now. Please remove.

Reply:
Thanks for the pickup.

Reply:
```suggestion
```